frontend/app/src/components/premium/GetPremiumButton.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+const premium = usePremium();
+
+const { mobile } = useDisplay();
+const { isPackaged, premiumURL, navigateToPremium } = useInterop();
+
+const { t } = useI18n();
```

Comment:
Since version 9 of vue-i18n is very strict with this let's move it to the top

Reply:
took care of it, we just have to keep it in mind going forward and we have to find an approach for the composables themselves because the composable doesn't work by default outside of the setup function